### ISSUE TITLE
Implement AlexNet Optimiser

### ISSUE THREAD

**Issue by samgd** [samgd/repaper#17]
Done Definition: Able to train AlexNet on random data using the optimizer outlined in the paper (SGD, momentum, weight decay, learning rate)
